### ISSUE TITLE
generate animated GIF or SVG

### ISSUE THREAD

**Issue by hofstef** [WPS/egon.io#55]
**Is your feature request related to a problem? Please describe.**
When I share a Domain Story as SVG or PNG with people who do not use the modeler, they have trouble to read (larger) stories in the right sequence.

**Describe the solution you'd like**
Additionally to the possibility of downloading a static picture, I would like to download an animated GIF. The animation should be slow enough so I can follow the story.

**Describe alternatives you've considered**
We could provide a "replay mode only" version of the modeler that runs in the web (e.g. www.wps.de/modeler/play) and that provides a super simple way for uploading and animating a .dst file. There might be other solutions that I have not considered yet.

**Comment by hschwentner** [comment]
I think we should animate SVGs instead of GIFs (and thus change the title to “generate animated SVG”). I experimented a bit with SVG's `<animate>` tag and think it does what we want. We could add something like the following to the sentences of a domain story:

```xml
<animate
    attributetype="XML"
    attributename="visibility" 
    from="hidden" 
    to="visible" 
    dur="1s">
</animate>
```

That would lead to an image where sentence after sentence will fade in.

Interesting sources:

- https://en.wikipedia.org/wiki/SVG_animation
- https://css-tricks.com/guide-svg-animations-smil/
- https://www.w3.org/TR/2001/REC-smil-animation-20010904/
- https://developer.mozilla.org/en-US/docs/Web/SVG/Element/animate?retiredLocale=de

**Comment by hschwentner** [comment]
Update: maybe `<set>` is enough. We could write something like the following on every domain story sentence and increment the seconds.

```xml
<set end="1s" attributeName="visibility" to="hidden"/>
```

**Comment by hofstef** [comment]
released with v2.2.0; some SVG tools do not support animations - see [https://egon.io/howto#exporting-domain-stories](https://egon.io/howto#exporting-domain-stories)
